security-common/src/main/java/org/codice/security/sign/Encoder.java

```diff
@@ -24,54 +26,56 @@
 
 public class Encoder {
 
-  private static final boolean GZIP_COMPATIBLE = true;
+    private static final boolean GZIP_COMPATIBLE = true;
 
-  private Encoder() {
-  }
+    private Encoder() {
```

Comment:
✏️ This doesn't appear to follow our google format standard for Java files.

Probably because it has not been added to the project. I'll rectify that.